src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -65,12 +65,17 @@ const defaultProps = {
 };
 
 function OptionRowLHN(props) {
+    const theme = useTheme();
+    const themeStyles = useThemeStyles();
```

Comment:
I'm wondering if we should still name this cost `styles` for consistency with the old naming convention. What are your thoughts on this?

Reply:
We can do that 👍